app/views/schools/withdrawn_requests/index.html.erb

```diff
@@ -63,7 +63,9 @@
         There are no withdrawn requests.
       </p>
     <% end %>
-
+    <p>
+      If you wish to download your withdrawn requests, please visit our <%= link_to "CSV export page", schools_csv_export_path %>.
```

Comment:
Change to 'Visit the CSV export page[link] to download your withdrawn requests.'